src/main/java/fr/zcraft/quartzlib/tools/items/ItemStackBuilder.java

```diff
@@ -652,37 +645,6 @@ public ItemStackBuilder data(short data)
         return this;
     }
 
-    /**
-     * Sets the dye color of the ItemStack. If the item is not colorable, an
-     * IllegalStateException will be thrown when making the item. Setting this
-     * value will override damage/data values.
-     *
-     * @param dye The dye color for the item.
-     *
-     * @return The current ItemStackBuilder instance, for methods chaining.
-     */
-    public ItemStackBuilder dye(DyeColor dye)
-    {
-        this.dye = dye;
-        return this;
-    }
-
-    /**
-     * Sets the skin name to use for this ItemStack. If the item is not of type
-     * {@link Material#PLAYER_HEAD}, an {@link IllegalStateException} will be
-     * thrown when making the item. Setting this will override the data value
-     * of the skull.
-     *
-     * @param owner The head's owner, implying the skin displayed on the head.
-     *
-     * @return The current ItemStackBuilder instance, for methods chaining.
-     */
-    public ItemStackBuilder head(String owner)
-    {
-        this.headOwner = owner;
-        return this;
-    }
```

Comment:
What about keeping this as as convenient method to set the material to `PLAYER_HEAD` and the skull meta to the right name?

Something like:

```java
final ItemStack head = new ItemStackBuilder().head("Pileurs").item();
```

Reply:
I think we'd want to keep the ItemStackBuilder as generic as possible, without getting involved in any specific Item, because `ItemMeta`s have pretty extensive APIs that we don't want to reproduce all in one place.

The `SkullMeta` API for example, now accepts both a player name and an `OfflinePlayer` (the player name version being actually deprecated), and using this helper doesn't shorten the code all that much. :/

I'm also not convinced heads are used that much more, compared to other items with more complex `ItemMeta`s, at least not enough to warrant an extra specific method.

Reply:
Okay, if we re-center the objective of this tool to be more generic than before, I'm fine with that.